source/TLS.rst

```diff
@@ -163,9 +192,64 @@ Supposing that your cluster name is ``my-cluster-name-rs0``, the instructions to
 
 	$ kubectl create secret generic my-cluster-name-ssl --from-file=tls.crt=client.pem --from-file=tls.key=client-key.pem --from-file=ca.crt=ca.pem --type=kubernetes.io/tls
 
+.. _tls.certs.update.check:
+
+Check your certificates for expiration
+================================================================================
+
+#. First, check the necessary secrets names (``my-cluster-name-ssl`` and 
+   ``my-cluster-name-ssl-internal`` by default):
+
+   .. code:: bash
+
+      $ kubectl get certificate
+
+   You will have the following response:
+
+   .. code:: text
+
+      NAME                           READY   SECRET                         AGE
+      my-cluster-name-ssl            True    my-cluster-name-ssl            49m
+      my-cluster-name-ssl-internal   True    my-cluster-name-ssl-internal   49m
+
+#. Optionally you can also check that the certificates issuer is up and running:
+
+   .. code:: bash
+
+      $ kubectl get issuer
+
+   The response should be as follows:
+
+   .. code:: text
+
+      NAME                       READY   AGE
+      my-cluster-name-psmdb-ca   True    61s
+
+#. Now use the following command to find out the certificates validity dates,
+   substituting Secrets names if necessary:
+
+   .. code:: bash
+
+      $ {
+        kubectl get secret/my-cluster-name-ssl-internal -o jsonpath='{.data.tls\.crt}' | base64 --decode | openssl x509 -inform pem -noout -text | grep "Not After"
+        kubectl get secret/my-cluster-name-ssl -o jsonpath='{.data.ca\.crt}' | base64 --decode | openssl x509 -inform pem -noout -text | grep "Not After"
+        }
```

Comment:
it might be a bit shorter 
`kubectl get secret/monitoring-ssl -o jsonpath='{.data.ca\.crt}' | base64 --decode | openssl x509 -noout -dates`